html/lankar.html

```diff
@@ -36,10 +36,50 @@
     <main>
 
         <ul class="breadcrumbs">
-            <li>Startsida / </li>
-            <li></li>
+            <li><a href="/index.html">Startsida</a> / </li>
+            <li>Länkar</li>
         </ul>
 
+        <h1>Länkar</h1>
+
+            <h2>Länkar till andra myndigheter och samarbetspartners</h2>
```

Comment:
Maybe wrap these in `<section>` and change `<h2>` to `<p>` to keep consistency between pages.